Assets/AppCenter/Plugins/iOS/Crashes/CrashesUnity.h

```diff
@@ -4,11 +4,12 @@
 #ifndef CRASHES_UNITY_H
 #define CRASHES_UNITY_H
 
+#import <Foundation/Foundation.h>
 #import <AppCenterCrashes/AppCenterCrashes.h>
+#import "MSException.h"
```

Comment:
No need to import it here. Forward declaration should be enough

Reply:
@jaelim-ms is that typical in our ios Code to just forward declare and not include?